apps/token/src/routes/claim/claim-flow.tsx

```diff
@@ -159,37 +156,34 @@ export const ClaimFlow = ({
           <div>
             <KeyValueTable>
               <KeyValueTableRow>
-                <th>{t('Connected Ethereum address')}</th>
-                <td>{truncateMiddle(address)}</td>
+                {t('Connected Ethereum address')}
```

Comment:
`<th>` and `<td>` tags have been moved inside KeyValueTableRow and tailwind styling has been applied to them.